src/contracts/xMETRIC.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+/// @notice We use solmate as it is more gas efficient thab OZ
+/// @notice https://github.com/Rari-Capital/solmate/blob/main/src/tokens/ERC20.sol
+/// @notice  https://github.com/Rari-Capital/solmate/blob/main/src/auth/Owned.sol
+
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {Owned} from "solmate/auth/owned.sol";
+
+contract xMETRIC is ERC20("xMETRIC", "xMETRIC", 18), Owned {
+    constructor(address _owner, uint256 initialSupply) Owned(_owner) {
+        _mint(msg.sender, initialSupply);
+    }
+
+    //------------------------------------------------------Overrides
+    function transfer(address to, uint256 amount)
+        public
+        override
+        transactor
+        returns (bool)
+    {
+        balanceOf[msg.sender] -= amount;
+
+        // Cannot overflow because the sum of all user
+        // balances can't exceed the max uint256 value.
+        unchecked {
```

Comment:
good point - I was reading about unchecked and thinking it could save on some gas costs and being able to utilize when it's safe in other contracts as well

Reply:
Agreed